.gitignore

```diff
@@ -0,0 +1 @@
+greynoise/__pycache__
```

Comment:
the __pycache__ folder is still a part of the PR. See the list of files below. Please remove the folder as well

Reply:
Okay, I believe this has been removed now.